src/analysisd/decoders/winevtchannel.c

```diff
@@ -376,330 +352,27 @@ int DecodeWinevt(Eventinfo *lf){
-                                case 14338:
-                                    subcategory = "Other Account Logon Events";
-                                    break;
-                                case 14339:
-                                    subcategory = "Kerberos Authentication Service";
-                                    break;
-                                default:
-                                    break;
-                            }
-                            break;
-                        default:
-                            break;
-                    }
-                    if (category) {
-                        cJSON_AddStringToObject(json_eventdata_in, "category", category);
-                    }
-                    if (subcategory) {
-                        cJSON_AddStringToObject(json_eventdata_in, "subcategory", subcategory);
-                    }
-
-                    os_free(categoryId);
-                    os_free(subcategoryId);
-                    os_free(filtered_categoryId);
-                    os_free(filtered_subcategoryId);
-                }
-            }
-
-            if (auditPolicyChangesId) {
-                int audit_split_n = 0;
-                char **audit_split;
-                char *audit_pol_changes = NULL;
-                char *audit_final_field = NULL;
-                int i = 0;
-
-                char * filtered_changes = wstr_replace(auditPolicyChangesId, "%%", "");
-                os_free(auditPolicyChangesId);
-
-                audit_split = OS_StrBreak(',', filtered_changes, 4);
-
-                while (audit_split[i]) {
-                    audit_split_n = strtol(audit_split[i], NULL, 10);
-
-                    switch (audit_split_n) {
-                        case 8448:
-                            wm_strcat(&audit_pol_changes, "Success removed", ',');
-                            break;
-                        case 8449:
-                            wm_strcat(&audit_pol_changes, "Success added", ',');
-                            break;
-                        case 8450:
-                            wm_strcat(&audit_pol_changes, "Failure removed", ',');
-                            break;
-                        case 8451:
-                            wm_strcat(&audit_pol_changes, "Failure added", ',');
-                            break;
-                        default:
-                            break;
-                    }
-
-                    i++;
-                }
-                audit_final_field = wstr_replace(audit_pol_changes, ",", ", ");
-                cJSON_AddStringToObject(json_eventdata_in, "auditPolicyChanges", audit_final_field);
-                os_free(filtered_changes);
-                os_free(audit_pol_changes);
-                os_free(audit_final_field);
-                free_strarray(audit_split);
             }
         }
         xml_init = 1;
     }
 
     find_msg = strstr(lf->log, "Message");
```

Comment:
Can the word `Message` appear in other fields of the event? In that case, the decoder would fail.